python_bindings/README.md

```diff
@@ -0,0 +1,33 @@
+# A python wrapper for easi
```

Comment:
Could you add this README to the RTD documentation? https://github.com/SeisSol/easi/tree/master/doc